src/serialize.rs

```diff
@@ -321,7 +317,7 @@ impl EGraph {
                             .extract_term(self, *value, &extractor, &mut termdag)
                             .expect("Extraction should be successful since extractor has been fully initialized");
 
-                    termdag.term_to_expr(&term).to_string()
+                    termdag.term_to_expr(&term, Span::Panic).to_string()
```

Comment:
Why not use the `span!()` macro here? Just trying to understand when you want panic vs that one.

Reply:
The `to_string` method ignores spans, so using `Span::Panic` is just a runtime check that the spans are in fact ignored.

In general, I think `Span::Panic` is useful when you know that the spans are not observable by outside code; i.e. I would avoid it if the function returned an `Expr`.